src/lib/rule-finder.js

```diff
@@ -103,6 +108,6 @@ function RuleFinder(specifiedFile, noCore) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, noCore) {
-  return new RuleFinder(specifiedFile, noCore);
+module.exports = function (specifiedFile, options = {}) {
```

Comment:
If you leave out the default value (`... (specifiedFile, options) {`) here, the tests will pass.  
(It's left to `RuleFinder` then, to assign a default value).